drivers/gpu/drm/bridge/cdns-mhdp.c

```diff
@@ -1434,7 +1456,6 @@ static int mhdp_link_training(struct cdns_mhdp_device *mhdp,
 			      unsigned int training_interval)
 {
 	u32 reg32;
-	union phy_configure_opts phy_cfg;
```

Comment:
This line removal looks a bit odd. Should it be part of "drm/bridge: cdns-mhdp: Remove extra phy configuration calls"?

Reply:
Yes. I forgot to remove this line in that patch.